docs/releases/4.0.txt

```diff
@@ -196,6 +196,9 @@ Forms
   :exc:`~django.core.exceptions.ValidationError` for the ``invalid_choice``
   error message. This allows custom error messages to use the ``%(value)s``
   placeholder.
+* :class:`~django.forms.RadioSelect` now only provides an ``id_for_label``
+  when rendering ``inputs``. Addresses accessibility issue where the field
+  label would point to the first input.
```

Comment:
I probably wouldn't understand what this was about without context. Maybe "would point to the first radio/checkbox"?